README.md

```diff
@@ -14,54 +13,39 @@ If you want to use it directly in a browser use a CDN like [Skypack](https://www
 ```javascript
 import urlJoin from 'url-join';
 
-const id = '42'
-const postUrl = urlJoin('/posts', id, '/metadata/');
-// => "/posts/42/metadata"
+// Create a Full URL
+const fullUrl = urlJoin('http://www.google.com', 'a', '/b/cd', '?foo=123', '&bar=456', '#heading-1');
+fullUrl.toString(); // 'http://www.google.com/a/b/cd?foo=123&bar=456#heading-1'
 ```
 ## You might not need this library
 
-Web Sandard APIs [URL][1] and [URLSearchParams][1] capable of URL operations except manipulating [URL pathnames][2].
-```
-Use this library to manipulate URL pathnames
-                                      👇
-   https  :// www.example.com /posts/42/metadata ?filter=author&page=1 #heading-1
-|-scheme-|    |---domain----| |----pathname----| |------search-------| |--hash--|
-```
+This library was originally created before the [URL][1] and [URLSearchParams][1] APIs were standardized and widely available in popular runtimes such as browsers and Node.js. Depending on your use-case you might want to consider using these standardized APIs over this library.
+
+```javascript
+const fullUrl = new URL('http://www.google.com');
 
-#### [`URL()`][1]
+fullUrl.pathname = '/a/b/cd';
+fullUrl.searchParams.append('foo', '123');
+fullUrl.searchParams.append('bar', '456');
+fullUrl.hash = 'heading-1';
 
-- Join origin to pathname, deduplicating `/` between origin and pathname
-```js
-let url = new URL("/wiki/Early_history_of_the_IRT_subway", "https://wikipedia.org/")
-// => "https://wikipedia.org/wiki/Early_history_of_the_IRT_subway"
+fullUrl.toString(); // 'http://www.google.com/a/b/cd?foo=123&bar=456#heading-1'
 ```
 
-- Set the url hash
-```js
-let url = new URL("https://example.com/")
-url.hash = "heading-1"
-// => "https://example.com/#heading-1"
-```
+### Caveats for Standard APIs
+There are a couple of caveats to take into account when utilizing the standard APIs. Firstly, a [URL][1] must always include a complete and valid base, which means specifying the scheme and domain name (e.g. http://example.com).
```

Comment:
I don't think we have to link to the MDN documentation here again, but let's use backticks for things that are part of the API:
```suggestion
There are a couple of caveats to take into account when utilizing the standard APIs. Firstly, a `URL` must always include a complete and valid base, which means specifying the scheme and domain name (e.g. http://example.com).
```